src/main/java/frc/robot/subsystems/drive/DriveSubsystem.java

```diff
@@ -500,8 +501,8 @@ public void offsetCurrentHeading(double offsetDegrees) {
     targetHeadingDegrees = getHeadingDegrees() + offsetDegrees;
   }
 
-  public void setZeroTargetHeading() {
-    targetHeadingDegrees = 0.0;
+  public void setForwardTargetHeading() {
+    targetHeadingDegrees = matchState.blue ? 0 : 180;
```

Comment:
Was there anything else we needed to change? Maybe a 180 deg rotation in one of the drive commands? Some of those comments might be off

Reply:
So I didn't see anything that seems like it would need to be adjusted, but I probably missed something.